sources/packages/backend/apps/api/src/app.students.module.ts

```diff
@@ -73,7 +77,7 @@ import { ATBCIntegrationModule } from "@sims/integrations/atbc-integration";
     StudentService,
     ProgramYearService,
     EducationProgramOfferingService,
-    DisbursementScheduleService,
+    DisbursementSchedulerService,
```

Comment:
I misunderstood this renaming from `DisbursementScheduleService` to `DisbursementSchedulerService` and it does not seem accurate.
The `DisbursementScheduleService` is the service for the entity `DisbursementSchedule` and it is not related to a `Scheduler`. Unless I am missing something it was wrongly renamed.

Reply:
oh... actually, I thought its `DisbursementSchedulerService`, and this is a typo. that's why I renamed it. reverting